src/osemgrep/core/Metrics_.ml

```diff
@@ -71,6 +71,14 @@ let default_payload =
         profilingTimes = [];
         maxMemoryBytes = None;
       };
+    extension =
```

Comment:
BTW could you try to rename Metrics_.ml to just Metrics.ml
or maybe it goes in conflict with some library we use that already have a Metrics.ml, but in any case
it's worth adding  a comment in the .ml to say why it's not called more simply just Metrics.ml

Reply:
I think @hannesm renamed it due to the usage of the `metrics` library - a clash occurs between the library and this module. I can add a note indeed.